src/sass/aiAssistant.scss

```
@@ -0,0 +1,61 @@
+}
+
+.messages-list {
+  .message-item {
+    .message-icon {
+      width: 40px;
+      height: 40px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      flex-shrink: 0;
+      background-color: lighten(#2094e8, 40%);
```

Stylelint (17.4.0)

[error] Expected color.adjust($color, $lightness: $amount) instead of lighten($color, $amount) (scss/no-global-function-names)

(scss/no-global-function-names)